homepage/models.py

```diff
@@ -80,10 +81,10 @@ def __str__(self):
 class Review(models.Model):
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
     user = models.ForeignKey(AppUser, on_delete=models.CASCADE)
-    rate = models.SmallIntegerField()
+    rate = models.SmallIntegerField(validators=[MinValueValidator(1), MaxValueValidator(5)])
     date = models.DateTimeField(default=timezone.now)
-    content = models.TextField(null=True, blank=True)
-    course_load = models.SmallIntegerField()
+    content = models.TextField(null=True, blank=True, max_length=200, validators=[MaxLengthValidator(200)])

```

Comment:
I my opinion 200 chars for content is not enough.
TextField is usually used for large texts that don't require max-length limitation, we can just limit the typing in the UI widget. Check out this [post](https://stackoverflow.com/questions/7354588/django-charfield-vs-textfield).
Anyway, if we want to limit it in the DB I think it should be at least 1000 chars.

Reply:
Good catch!
I'll remove the ```MaxLengthValidator```